extension/src/prototypes/ui-components/PropertyParameterItem.tsx

```
@@ -203,25 +215,27 @@ const Property: FC<{
   property: { id, name, values, path: defaultPath },
   level,
   path,
   featureType,
   ancestorsFeatureType,
+  version,
 }) => {
   const isPrimitive = ["string", "number"].includes(typeof values[0]);
   const hasAncestors = !!path?.includes(ancestorsKey);
   const actualName = `${hasAncestors ? ancestorsFeatureType : featureType}_${[
     ...((hasAncestors ? path?.filter(p => p !== ancestorsKey) : path) ?? []),
     ...(isNaN(Number(name)) ? [name] : []),
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)